### PR TITLE
scuzz: make some fixes

### PR DESCRIPTION
**Pull request by hugelgupf**
This never worked... most likely due to the ioctl error, at least, where p.cmd was passed instead of the SG_IO ioctl req.

Added a QEMU Test for the Identify command.

### CODE REVIEW

File: pkg/scuzz/sg_linux.go [hunk 1]
```diff
@@ -144,10 +144,20 @@ func NewSGDisk(n string, opt ...SGDiskOpt) (Disk, error) {
 	return s, nil
 }
 
+const SAT_ATA_PASSTHROUGH_12 = 0xa1
+
 // genCommandDataBlock creates a Command and Data Block used by
 // Linux SGDisk.
 func (p *packet) genCommandDataBlock() {
-	p.command[0] = ata16
+	p.command[0] = SAT_ATA_PASSTHROUGH_12
```
[the remark this answers is not in the document]
Comment (hugelgupf) [reply pkg/scuzz/sg_linux.go:152]: You were right. The problem was merely the ioctl number -- it needed to be SG_IO, not p.cmd.

[the remark this answers is not in the document]
Comment (hugelgupf) [reply pkg/scuzz/sg_linux.go:152]: reverted the ata12 change.

File: pkg/scuzz/sg_linux.go [hunk 2]
```diff
@@ -263,9 +275,9 @@ func (s *SGDisk) operate(p *packet) error {
 		return err
 	}
 	// the drive must be ata48. The status should show that even if we did not issue an ata48 command.
-	if err := w.mustLBA(); err != nil {
+	/*if err := w.mustLBA(); err != nil {
```
[the remark this answers is not in the document]
Comment (hugelgupf) [reply pkg/scuzz/sg_linux.go:277]: I don't fully understand yet, but despite the fact that I've gone back to ata16, this mustLBA thing still fails on the disks I'm trying it on.

**Comment by hugelgupf** [comment]
Made some fixes and added a QEMU test for SGDisk.Identify.

File: pkg/scuzz/sg_linux.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (rminnich) [reply pkg/scuzz/sg_linux.go:285]: as long as everything works anyway, we can chalk it up to another ATA fail.

**Pull request merged**